### PR TITLE
Add beta of BitBar

### PR DESCRIPTION
**Pull request by tresni**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
You are missing [the appcast](https://github.com/caskroom/homebrew-cask/blob/f51a615c0c2ce088f1742a09e760d9080b009b5e/Casks/bitbar.rb#L6L7).

File: Casks/bitbar-beta.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'bitbar-beta' do
+  version '1.5.0-beta1'
+  sha256 '4ef9a1e113b31f3b1f6902bca9c1fca7e1d5ab02ca3159f15fd079fef8626668'
+
+  url 'https://github.com/matryer/bitbar/releases/download/v1.5.0-beta1/BitBar-v1.5-beta1.zip'
```
Comment (adidalal) [inline Casks/bitbar-beta.rb]: Use string interpolation here, ie `#{version}`

Comment (vitorgalvao) [inline Casks/bitbar-beta.rb]: @adidalal It’s difficult to do in this case, on account of having `.0` in one part but not the other.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in d8ab521fae0b71959042552ff3c7f38a617afccc. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by tresni** [comment]
@vitorgalvao Thanks for doing that, been a busy week for me so far and hadn't gotten the time.

**Pull request closed without merging**